### PR TITLE
[DOCS-389] Browser log NPM

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

Adds NPM instructions for browser log collection

### Motivation
The NPM package is now available.

### Preview link

* https://docs-staging.datadoghq.com/gus/browser-log-npm/logs/log_collection/javascript/?tab=us

### CODE REVIEW

File: content/en/logs/log_collection/javascript.md [hunk 1]
```diff
@@ -32,17 +26,47 @@ With the `datadog-logs` library, you can send logs directly to Datadog from JS c
 * Record real client IP addresses and user agents.
 * Optimized network usage with automatic bulk posts.
 
-## Get a Client Token
+## Setup
+
+1. **Get a Datadog Client Token**: For security reasons, [API keys][1] cannot be used to configure the `datadog-logs` library, as they would be exposed client-side in the JavaScript code. To collect logs from web browsers, a [client token][2] must be used. For more information about setting up a client tolken, see the [Client tokens documentation][2].
```
Comment (sarina-dd) [inline content/en/logs/log_collection/javascript.md]:
```suggestion
1. **Get a Datadog Client Token**: For security reasons, [API keys][1] cannot be used to configure the `datadog-logs` library, as they would be exposed client-side in the JavaScript code. To collect logs from web browsers, a [client token][2] must be used. For more information about setting up a client tolken, see the [client token documentation][2].
```

**Comment by sarina-dd** [comment]
Under the bundle setup header, there are missing HTML tags from the EU example.

File: content/en/logs/log_collection/javascript.md [hunk 2]
```diff
@@ -55,75 +79,78 @@ In order to not miss any logs or errors, you should load and configure the libra
...
 ```
 
+**Note**: The `window.DD_LOGS` check is used to prevent issues if a loading failure occurs with the library.
+
+{{% /tab %}}
+{{< /tabs >}}
+
 This gives the following result:
 
 ```json
```
Comment (NBParis) [inline content/en/logs/log_collection/javascript.md]: @hdelaby I believe that now it adds view.url and not http.url by default, is that correct?

Comment (hdelaby) [reply content/en/logs/log_collection/javascript.md]: Yes indeed
Image 2020-01-08 at 1 18 15 PM

Comment (l0k0ms) [reply content/en/logs/log_collection/javascript.md]: fixed !

**Pull request merged**